tests/ZiraLink.Api.Application.UnitTests/Services/CustomerServiceTest.cs

```diff
@@ -42,43 +48,39 @@ public async Task GetCustomerByExternalId_WhenEverythingIsOk_ShouldHasData(strin
         public async Task GetCustomerByExternalId_WhenExternalIdIsNotExist_ShouldHasData(string externalId)
         {
             //Arrange
-            var mockConfiguration = new Mock<IConfiguration>(); 
-            mockConfiguration.Setup(m => m["ZIRALINK_URL_IDS"]).Returns("https://ids.ziralink.local:5001");
+            var mockHttpTools = new Mock<IHttpTools>();
 
-            CustomerService customerService = new CustomerService(_testTools.AppMemoryDbContext, mockConfiguration.Object);
+            CustomerService customerService = new CustomerService(_testTools.AppMemoryDbContext, mockHttpTools.Object);
 
             var exception = await Assert.ThrowsAsync<NotFoundException>(() => customerService.GetCustomerByExternalIdAsync(externalId, CancellationToken.None));
             Assert.Equal("Customer", exception.Message);
         }
         #endregion
 
         #region GetAll
-        [Fact] 
+        [Fact]
         public async Task GetAll_ShouldHasData()
         {
             //Arrange
-            var mockConfiguration = new Mock<IConfiguration>();
-            mockConfiguration.Setup(m => m["ZIRALINK_URL_IDS"]).Returns("https://ids.ziralink.local:5001");
+            var mockHttpTools = new Mock<IHttpTools>();
 
-            CustomerService customerService = new CustomerService(_testTools.AppMemoryDbContext, mockConfiguration.Object);
+            CustomerService customerService = new CustomerService(_testTools.AppMemoryDbContext, mockHttpTools.Object);
 
-            var response = await customerService.GetAllAsync(CancellationToken.None); 
-            Assert.True(response is List<Customer>); 
+            var response = await customerService.GetAllAsync(CancellationToken.None);
+            Assert.True(response is List<Customer>);
             Assert.True(response.Any());
-        } 
+        }
         #endregion
 
-        #region CreateLocallyAsync
+        #region CreateCustomerLocally
         [Theory]
         [InlineData("10", "TestUser", "TestEmail@email.com", "Test", "Testi")]
-        public async Task CreateLocallyAsync_WhenEverythingIsOk_ShouldBeSucceeded(string externalId, string username, string email, string name, string family)
+        public async Task CreateCustomerLocally_WhenEverythingIsOk_ShouldBeSucceeded(string externalId, string username, string email, string name, string family)
```

Comment:
I would suggest keeping the first part `CreateLocallyAsync` as it is the method's name

Reply:
Done.